patches/tModLoader/Terraria/GameContent/TownNPCProfiles.cs.patch

```diff
@@ -0,0 +1,60 @@
+--- src/TerrariaNetCore/Terraria/GameContent/TownNPCProfiles.cs
++++ src/tModLoader/Terraria/GameContent/TownNPCProfiles.cs
+@@ -1,4 +_,6 @@
+ using System.Collections.Generic;
++using Terraria.ID;
++using Terraria.ModLoader;
+ 
+ namespace Terraria.GameContent
+ {
+@@ -29,32 +_,48 @@
+ 			43,
+ 			44
+ 		};
++		// does Old Man count? should he? need to reference vanilla or ask someone who's done so already     -thomas
```

Comment:
Is this Todo sorted, or is this still an open action?

Reply:
I have no idea, but he didn't show up there even though Clothier (his non-cursed variant) did, so I'll assume Old Man isn't included in this and strike this todo accordingly